Unable to build tensorflow benchmark tool for android: undefined reference to `vtable for tflite::gpu::cl::SpaceToDepth'

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04.4 LTS
- TensorFlow installed from (source or binary): Source
- TensorFlow version:  Build from: git clone --recurse-submodules https://github.com/tensorflow/tensorflow.git
- Python version: 3.6.9
- Bazel version (if compiling from source): 2.0.0
- GCC/Compiler version (if compiling from source):gcc (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0

Here is the bazel workspace configuration:-

> WARNING: Running Bazel server needs to be killed, because the startup options are different.
> WARNING: ignoring LD_PRELOAD in environment.
> You have bazel 2.0.0 installed.
> Please specify the location of python. [Default is /usr/bin/python3]: 
> 
> 
> Found possible Python library paths:
>   /usr/lib/python3/dist-packages
>   /usr/local/lib/python3.6/dist-packages
> Please input the desired Python library path to use.  Default is [/usr/lib/python3/dist-packages]
> 
> Do you wish to build TensorFlow with XLA JIT support? [Y/n]: n
> No XLA JIT support will be enabled for TensorFlow.
> 
> Do you wish to build TensorFlow with OpenCL SYCL support? [y/N]: N
> No OpenCL SYCL support will be enabled for TensorFlow.
> 
> Do you wish to build TensorFlow with ROCm support? [y/N]: N
> No ROCm support will be enabled for TensorFlow.
> 
> Do you wish to build TensorFlow with CUDA support? [y/N]: N
> No CUDA support will be enabled for TensorFlow.
> 
> Do you wish to download a fresh release of clang? (Experimental) [y/N]: N
> Clang will not be downloaded.
> 
> Please specify optimization flags to use during compilation when bazel option "--config=opt" is specified [Default is -march=native -Wno-sign-compare]: 
> 
> 
> Would you like to interactively configure ./WORKSPACE for Android builds? [y/N]: y
> Searching for NDK and SDK installations.
> 
> Please specify the home path of the Android NDK to use. [Default is /root/Android/Sdk/ndk-bundle]: /content/tensorflow/android-ndk-r18b
> 
> 
> Please specify the (min) Android NDK API level to use. [Available levels: ['16', '17', '18', '19', '21', '22', '23', '24', '26', '27', '28']] [Default is 21]: 
> 
> 
> Please specify the home path of the Android SDK to use. [Default is /root/Android/Sdk]: /content/Sdk
> 
> 
> Please specify the Android SDK API level to use. [Available levels: ['28']] [Default is 28]: 
> 
> 
> Please specify an Android build tools version to use. [Available versions: ['29.0.0']] [Default is 29.0.0]: 
> 
> 
> Preconfigured Bazel build configs. You can use any of the below by adding "--config=<>" to your build command. See .bazelrc for more details.
> 	--config=mkl         	# Build with MKL support.
> 	--config=monolithic  	# Config for mostly static monolithic build.
> 	--config=ngraph      	# Build with Intel nGraph support.
> 	--config=numa        	# Build with NUMA support.
> 	--config=dynamic_kernels	# (Experimental) Build kernels into separate shared objects.
> 	--config=v2          	# Build TensorFlow 2.x instead of 1.x.
> Preconfigured Bazel build configs to DISABLE default on features:
> 	--config=noaws       	# Disable AWS S3 filesystem support.
> 	--config=nogcp       	# Disable GCP support.
> 	--config=nohdfs      	# Disable HDFS support.
> 	--config=nonccl      	# Disable NVIDIA NCCL support.
> Configuration finished`

`
**Describe the problem**
Unable to compile Tflite benchmark tool for android_arm64 platform.It gives the following error:-

```
!bazel build -c opt \
  --config=android_arm64 \
  tensorflow/lite/tools/benchmark:benchmark_model
```
(Also tried argument options plain arm, cxx-opt 14 etc..)
> Compiling tensorflow/lite/delegates/gpu/common/model_builder.cc; 4s local
> ERROR: /content/tensorflow/tensorflow/lite/tools/benchmark/BUILD:22:1: Linking of rule '//tensorflow/lite/tools/benchmark:benchmark_model' failed (Exit 1)
> bazel-out/arm64-v8a-opt/bin/tensorflow/lite/delegates/gpu/cl/selectors/libsimple_selectors.a(simple_selectors.o): In function `tflite::gpu::cl::SelectSpaceToDepth(tflite::gpu::SpaceToDepthAttributes const&, tflite::gpu::cl::OperationDef const&, std::__ndk1::unique_ptr<tflite::gpu::cl::GPUOperation, std::__ndk1::default_delete<tflite::gpu::cl::GPUOperation> >*)':
> /proc/self/cwd/tensorflow/lite/delegates/gpu/cl/selectors/simple_selectors.cc:132: undefined reference to `tflite::gpu::cl::CreateSpaceToDepth(tflite::gpu::cl::OperationDef const&, tflite::gpu::SpaceToDepthAttributes const&)'
> bazel-out/arm64-v8a-opt/bin/tensorflow/lite/delegates/gpu/cl/selectors/libsimple_selectors.a(simple_selectors.o): In function `std::__ndk1::__unique_if<tflite::gpu::cl::SpaceToDepth>::__unique_single std::__ndk1::make_unique<tflite::gpu::cl::SpaceToDepth, tflite::gpu::cl::SpaceToDepth>(tflite::gpu::cl::SpaceToDepth&&)':
> /proc/self/cwd/external/androidndk/ndk/sources/cxx-stl/llvm-libc++/include/memory:3114: undefined reference to `tflite::gpu::cl::SpaceToDepth::SpaceToDepth(tflite::gpu::cl::SpaceToDepth&&)'
> bazel-out/arm64-v8a-opt/bin/tensorflow/lite/delegates/gpu/cl/selectors/libsimple_selectors.a(simple_selectors.o): In function `~SpaceToDepth':
> /proc/self/cwd/./tensorflow/lite/delegates/gpu/cl/kernels/space_to_depth.h:29: undefined reference to `vtable for tflite::gpu::cl::SpaceToDepth'
> /proc/self/cwd/./tensorflow/lite/delegates/gpu/cl/kernels/space_to_depth.h:29: undefined reference to `vtable for tflite::gpu::cl::SpaceToDepth'
> /proc/self/cwd/./tensorflow/lite/delegates/gpu/cl/kernels/space_to_depth.h:29: undefined reference to `vtable for tflite::gpu::cl::SpaceToDepth'
> /proc/self/cwd/./tensorflow/lite/delegates/gpu/cl/kernels/space_to_depth.h:29: undefined reference to `vtable for tflite::gpu::cl::SpaceToDepth'
> clang: error: linker command failed with exit code 1 (use -v to see invocation)
> Target //tensorflow/lite/tools/benchmark:benchmark_model failed to build
> Use --verbose_failures to see the command lines of failed build steps.
> INFO: Elapsed time: 833.996s, Critical Path: 35.28s
> INFO: 1497 processes: 1497 local.
> FAILED: Build did NOT complete successfully

**Provide the exact sequence of commands / steps that you executed before running into the problem**
 Followed the official GitHub: https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/tools/benchmark

I tried with NDK 14 and it gave error related to string parsing during compilation. In  18 and 20 NDK versions it gives error related to SpacetoBatch. I tried the compilation in google colab using command line tools for sdk manager for android.

--- a/tensorflow/lite/delegates/gpu/cl/selectors/BUILD
+++ b/tensorflow/lite/delegates/gpu/cl/selectors/BUILD
@@ -119,6 +119,7 @@ cc_library(
         "//tensorflow/lite/delegates/gpu/cl/kernels:resize",
         "//tensorflow/lite/delegates/gpu/cl/kernels:softmax",
         "//tensorflow/lite/delegates/gpu/cl/kernels:softmax1x1",
+        "//tensorflow/lite/delegates/gpu/cl/kernels:space_to_depth",
         "//tensorflow/lite/delegates/gpu/cl/kernels:strided_slice",
         "//tensorflow/lite/delegates/gpu/cl/kernels:transpose",
         "//tensorflow/lite/delegates/gpu/common:operations",

